src/ClientRequestContext.h

```diff
@@ -80,8 +80,13 @@ class ClientRequestContext : public RefCountable
 #if USE_OPENSSL
     bool sslBumpCheckDone = false;
 #endif
-    ErrorState *error = nullptr; ///< saved error page for centralized/delayed processing
+
     bool readNextRequest = false; ///< whether Squid should read after error handling
+    ErrorState *error = nullptr; ///< saved error page for centralized/delayed processing
```

Comment:
I'm happy to approve the PR once this trivial request is satisfied

Reply:
> I'm happy to approve the PR once this trivial request is satisfied

Francesco, this comment was not requesting a PR change but explaining an existing (and seemingly out of scope) change. In other words, the data member move discussed in my comment has been implemented already (branch commit eddb0bfe08e896c3831f647ea8c427c228328212). I left that "I do not insist on this change" GitHub comment just in case others want to _undo_ that change.

Sorry for the confusion. In the future, I will try to remember to explicitly mark similar comments as comments that explain existing changes rather than requesting future changes.

Reply:
> Sorry for the confusion

Nothing to be sorry about, I should have read the code more carefully